app/uk/gov/hmrc/gform/dms/DmsSubmissionAlgebra.scala

```diff
@@ -58,14 +69,38 @@ class DmsSubmissionService[F[_]](
       pdfDoc = documentLoader(pdfBytes)
       pdfSummary = PdfSummary(pdfDoc.getNumberOfPages.toLong, pdfBytes)
       _ = pdfDoc.close()
-      submission = DmsSubmissionService.createSubmission(metadata, envId, LocalDateTime.now(clock), Attachments.empty)
+      submission = DmsSubmissionService.createSubmission(metadata, envId, LocalDateTime.now(clock), 0)
       summaries = PdfAndXmlSummaries(pdfSummary)
       dmsSubmission = DmsSubmissionService.createDmsSubmission(metadata)
       _ <- fileUpload.submitEnvelope(submission, summaries, dmsSubmission)
     } yield envId
   }
 
+  override def submitPdfToDmsWithAttachments(html: String, attachments: List[FileAttachment], metaData: DmsMetadata)(
```

Comment:
Implement `submitToDms` in terms of `submitPdfToDmsWithAttachments `.

Reply:
I'll take a look.

Reply:
don't you mean the other way around? 
i.e. implement `submitPdfToDmsWithAttachments` in terms of `submitToDms`
i.e. leverage the html -> pdf functionality in `submitToDms` inside `submitPdfToDmsWithAttachments`

Reply:
fixed